change/@fluentui-react-native-button-2020-09-01-13-03-55-a11yState.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
nit, shouldn't this be consistent with the checkbox and radiogroup change types?

Reply:
Yes, I just updated it. Thanks!